Imgsmall.java

```diff
@@ -0,0 +1,34 @@
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.awt.image.RenderedImage;
+import java.io.File;
+import java.net.URL;
+
+public class Imgsmall {
+  public static void main(String[] args) throws Exception{
+    //get url object
+    URL url = new URL(args[0]);
+    float small = (100-Integer.parseInt(args[1]))/100;
```

Comment:
Instead of calling it small, call it resizeRatio or resizeBy or something like that descriptive.

Reply:
100.0 instead of 100